src/storage/store/var_list_node_column.cpp

```diff
@@ -53,9 +53,12 @@ void VarListNodeColumn::scanInternal(transaction::Transaction* transaction,
 
 void VarListNodeColumn::lookupValue(transaction::Transaction* transaction,
     common::offset_t nodeOffset, common::ValueVector* resultVector, uint32_t posInVector) {
-    auto listOffset = readListOffsetInStorage(transaction, nodeOffset);
-    auto length = readListOffsetInStorage(transaction, nodeOffset + 1) -
-                  readListOffsetInStorage(transaction, nodeOffset);
+    auto nodeGroupIdx = getNodeGroupIdxFromNodeOffset(nodeOffset);
+    auto nodeOffsetInGroup =
+        nodeOffset - (nodeGroupIdx << common::StorageConstants::NODE_GROUP_SIZE_LOG2);
```

Comment:
Use `StorageUtils::getStartOffsetForNodeGroup` for `nodeGroupIdx << common::StorageConstants::NODE_GROUP_SIZE_LOG2`.